receiver/k8sclusterreceiver/collection/metadata.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collection
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver/utils"
+)
+
+const (
+	// Keys for K8s properties
+	k8sKeyWorkLoad     = "k8s.workload"
```

Comment:
This should be "k8s.workload.kind"? Also not sure where is this used, can you point me to where do we use this?

Based on the usage in this file this seems to be unnecessary, you do have the kind/type in the `resourceIDKey`.

Reply:
Yes, `k8s.worload.kind` is better. 

Will ping you offline with some details about these workload properties.